pkg/kubelet/sysctl/namespace.go

```diff
@@ -48,6 +48,9 @@ var prefixNamespaces = map[string]Namespace{
 // NamespacedBy returns the namespace of the Linux kernel for a sysctl, or
 // unknownNamespace if the sysctl is not known to be namespaced.
 func NamespacedBy(val string) Namespace {
+	// The '/' separator is also accepted in place of a '.' in sysctl variable.
+	// More info: https://man7.org/linux/man-pages/man8/sysctl.8.html
+	val = strings.Replace(val, "/", ".", -1)
```

Comment:
at first glance, this doesn't seem to follow the logic described where the first separator determines how the remaining separators are handled

Reply:
https://github.com/kubernetes/kubernetes/blob/4748bb04b61f8717641913a0ff3a183ad28e64c9/pkg/kubelet/sysctl/namespace.go#L37-L47

https://github.com/kubernetes/kubernetes/blob/4748bb04b61f8717641913a0ff3a183ad28e64c9/pkg/security/podsecuritypolicy/sysctl/mustmatchpatterns.go#L32-L40

The namespace listed contains only dots. If the enumeration contains sysctl for slashes and dots, that would be too many items. Any better suggestion？